packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -11,13 +11,16 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/gestures.dart';
 import 'package:flutter/semantics.dart';
 import 'package:flutter/services.dart';
+import 'package:flutter/cupertino.dart' show CupertinoColors;
```

Comment:
this is a layering violation. This file can't know anything about Material or iOS.

Reply:
Ok, so then I should add a field for the color whose default is set to the color I have?